samples/bluetooth/iso_combined_bis_and_cis/README.rst

```diff
@@ -0,0 +1,120 @@
+.. _bluetooth_isochronous_combined_bis_and_cis:
+
+Bluetooth: ISO Combined BIS and CIS
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Combined BIS and CIS sample demonstrates capability of
+a Controller to support both Broadcast Isochronous Stream (BIS) and Connected Isochronous Stream (CIS) and
+can be used to learn ISO-specific APIs to create stream, receive and send data over either CIS or BIS.
+Note: the sample uses the simplest way to send data over ISO without using timestamps through a single stream,
+therefore it doesn't demonstrate any capabilities of synchronization multiple streams.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+You can use any of the development kits listed above and mix different development kits.
+
+.. include:: /includes/hci_ipc_overlay.txt
+
+The sample also requires a connection to a computer with a serial terminal |ANSI| for each of the development kits.
+
+Overview
+********
+
+The sample demonstrates a device combining CIS Central and BIS Source.
+As a CIS Central, it only implements receiving of data from CIS Peripheral.
+In order to provide the best performance and minimize number of conflicts between CIS, BIS and ACL packets the following measures are taken:
+* The same ISO Interval is used for both CIS and BIS. The value is set to 10 ms by default.
+* ACL Interval is set to integer multiple of ISO Interval to avoid collisions. The value is set to 20 ms by default.
+* Periodic advertising of BIS is set to the same value as ACL Interval.
+* Number of retransmissions is set to a high value - 10, to ensure that packets are retransmitted in case of collisions.
+* The sample as Central rejects incoming Connection Update requests to avoid changing the ACL Interval value.
+* In addition the following optional SDC-specific configurations are used:
+   * CONFIG_BT_CTLR_SDC_CENTRAL_ACL_EVENT_SPACING_DEFAULT is set to 10 ms to ensure that periodic advertising packet is set in between ACL packets.
```

Comment:
Use :kconfig:option: to get links to the KConfigs

Reply:
Thanks, I added :kconfig:option::)